Source/Support/DispatchSource+Fisticuffs.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DispatchSource+Fisticuffs.swift
+//  Fisticuffs
+//
+//  Created by Maksym Korytko on 2016-12-22.
+//  Copyright © 2016 theScore. All rights reserved.
+//
+
+import Foundation
+
+extension DispatchSource {
+    /**
+     Creates a single use timer source and resumes it.
+     Once the timer fires, it is automatically invalidated.
+     - parameters:
+        - queue: The dispatch queue that should be used to enque the handler.
+        - interval: Timer interval since now when the handler should be invoked.
+        - leeway: The handler may be called at any time between its scheduled time + leeway.
+        - handler: A function that should be called when the timer fires.
+     
+     - returns: An instance of the DispatchSourceTimer protocol.
+                The returned timer will be in the resumed state.
+     */
+    class func makeScheduledOneshotTimer(
```

Comment:
Ah, well if `DispatchSource` is already using "make" you might as well stick to that then 👍. I think the extra "source" suffix seems redundant but it could be good to stick with what they're already doing in that Framework… your call! 🙂